files/build_templates/sonic_debian_extension.j2

```diff
@@ -144,9 +144,9 @@ sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip3 install blkinf
 sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip3 install ipaddr
 
 # Install Python module for grpcio and grpcio-toole
-if [[ $CONFIGURED_ARCH == amd64 ]]; then
```

Comment:
@saiarcot895, here grpcio tools only installed for amd64 so I have added it for arm64 as well, and modified version to `1.58.0`.